content/uc-doc/installation/install-system.md

```diff
@@ -4,8 +4,21 @@ title: Installing the System
 
 ## Requirements
 
-- Memory: 2 GiB of memory is a tight minimum
-- Storage: 2.5 GiB of storage is a very tight minimum, 8 GiB is comfortable
+### Trying it out
+
+Here are the minimal resources needed for trying out Wazo Platform in a playground server:
+
+- CPU: 1 CPU is enough for playground
+- Memory: 2 GiB of memory is a tight minimum for playground
+- Storage: 8 GiB is comfortable for playground, except if you plan to use large-sized phone firmwares
```

Comment:
I'm not sure we need to repeat "playground" for every entry. I also think "sandbox" or test server? might be better terms. And "firmware" shouldn't have an "s". You can say large firmware files, but not "firmwares".

Reply:
Thanks for the suggestions and tips :)
I'll keep the repetition of "test server" so that those requirements can't possibly be mistaken for production requirements.